TaskWithMemoryAllocation/TaskWithMemoryAllocation.cpp

```diff
@@ -0,0 +1,38 @@
+﻿#include <iostream>
+#include <string>
+
+int main(int argc, char* argv[])
+{
+	if (argc == 1)
+	{
+		std::cout << "No command line arguments " << std::endl;
+		return 1;
+	}
+	std::cout << argv[1] << std::endl;
+
+	unsigned long long num;
+	try
+	{
+		num = std::stoull(argv[1]);
+	}
+	catch (const std::invalid_argument& e)
+	{
+		std::cout << "Allocation failed words: " << e.what() << std::endl;
```

Comment:
Just a minor thing. The message should be "Parse failed: ".

Reply:
Fixed